docker/dev/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Base image
+FROM node:18-alpine
+
+# Set the working directory
+WORKDIR /app
+
+# Copy package.json and package-lock.json
+COPY ../../package.json .
```

Comment:
this would change to look like 
```suggestion
COPY ./package.json .
```

Reply:
that's right but according to project folder structure that's how it must be followed

Reply:
okay